client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -94,6 +105,17 @@ const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedC
 				{ translate( 'Pick an email solution' ) }
 			</h1>
 
+			<div className="email-providers-stacked-comparison__how-they-compare">
+				{ translate( 'Not sure how to start?' ) }
+				<Button
+					borderless
+					className="email-providers-stacked-comparison__how-they-compare-link"
+					onClick={ () => page( emailManagementInDepthComparison( siteName, selectedDomainName ) ) }
+				>
+					{ translate( 'See how they compare.' ) }
+				</Button>
```

Comment:
On the mockup this is just a link, why do we need this `<Button>` component? As far as I know, we can achieve the same thing with an HTML anchor, and without additional CSS styles.